server/Makefile

```diff
@@ -126,3 +126,12 @@ docker-build:
 .PHONY: deploy
 deploy: kustomize yq
 	YQ="$(YQ)" KUSTOMIZE="$(KUSTOMIZE)" ./hack/deploy.sh "$(NAMESPACE)" "$(IMG)"
+
+.PHONY: package
+package: kustomize yq
+	JWKS_URL="https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/certs" \
+			 TOOLCHAIN_HOST="toolchain-host-operator" \
+			 YQ="$(YQ)" \
+			 KUSTOMIZE="$(KUSTOMIZE)" \
+			 MANIFEST_TARBALL="$$(realpath ./server.tar.gz)" \
```

Comment:
I'd propose to use the same approach used in `operator/Makefile`, i.e. declare `MANIFEST_TARBALL` as a Make variable at the top of the file and to default it to `out/server.tar.gz`

Reply:
Fixed in 1177ea0.